app/services/service_catalog/rbac_access.rb

```diff
@@ -0,0 +1,32 @@
+module ServiceCatalog
+  class RBACAccess
```

Comment:
These general rbac classes should not live under the `service_catalog` folder as it has nothing directly to do with it.  It should probably live in a `rbac` folder or even over in `lib/rbac`.